client-tools/Shell/v8-deadline.cpp

```diff
@@ -309,4 +309,8 @@ void TRI_InitV8Deadline(v8::Isolate* isolate) {
   TRI_AddGlobalFunctionVocbase(
       isolate, TRI_V8_ASCII_STRING(isolate, "SYS_INTERRUPT_TO_DEADLINE"),
       JS_RegisterExecutionDeadlineInterruptHandler);
+  if (timeout != 0) {
+    std::lock_guard mutex{singletonDeadlineMutex};
+    executionDeadline = TRI_microtime() + (timeout * 1000) / 1000;
```

Comment:
timeout * 1000 / 1000? 
Precedence of operators.
Also, I only see this being set. Where is the effect?

Reply:
dang that bracket should be up front. fixed.
Its a global variable since its shared among several threads and operated under the `singletonDeadlineMutex`  lock.